### PR TITLE
api_user_co_access

### PR DESCRIPTION
**Pull request by ioigoume**
Api user co level access

```sql
alter table cm_api_users add column co_id integer references cm_cos(id);
alter table cm_api_users add column privileged boolean;
alter table cm_api_users add column status varchar(2);
alter table cm_api_users add column valid_from timestamp;
alter table cm_api_users add column valid_through timestamp;
alter table cm_api_users add column remote_ip varchar(80);
```

### CODE REVIEW

File: app/Controller/AppController.php [hunk 1]
```diff
@@ -213,11 +258,11 @@ public function beforeFilter() {
       // See if we've collected it from the browser in a previous page load. Otherwise
       // use the system default. If the user set a preferred timezone, we'll catch that below.
       
-      if(!empty($_COOKIE['cm_registry_tz_auto'])) {
+      if(!empty($_COOKIE['dev_co_registry_tz_auto'])) {
```
Comment (lionick) [inline app/Controller/AppController.php]: This should not be changed.

File: app/Controller/AppController.php [hunk 2]
```diff
@@ -213,11 +258,11 @@ public function beforeFilter() {
...
         // We have an auto-detected timezone from a previous page render from the browser.
         // Adjust the default timezone. Actually, don't we want to always record times in UTC.
         //        date_default_timezone_set($_COOKIE['cm_registry_tz_auto']);
-        $this->set('vv_tz', $_COOKIE['cm_registry_tz_auto']);
+        $this->set('vv_tz', $_COOKIE['dev_co_registry_tz_auto']);
```
Comment (lionick) [inline app/Controller/AppController.php]: This should not be changed.

**Review by lionick — CHANGES_REQUESTED** [review]

File: app/Controller/AppController.php (hunk 1 above)
Comment (ioigoume) [reply app/Controller/AppController.php]: Done

File: app/Controller/AppController.php (hunk 2 above)
Comment (ioigoume) [reply app/Controller/AppController.php]: Done

**Review by lionick — APPROVED** [review]
LGTM

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**